conf/messages.en

```diff
@@ -808,42 +796,40 @@ title.niue=Niue Island
 title.norfolk_island=Norfolk Island
 title.northern_mariana_islands=Northern Mariana Islands
 title.norway=Norway (including Svalbard Archipelago and Jan Mayen Island)
+title.north_macedonia=North Macedonia
 title.occupied_palestinian_territory=Occupied Palestinian Territory (West Bank (including East Jerusalem) and Gaza Strip)
 title.oman=Oman (formerly Muscat and Oman)
 title.pakistan=Pakistan
-title.palra_island=Palra Island
 title.palau=Palau
+title.title.palmyra_atoll=Palmyra Atoll
 title.panama=Panama (including the former Canal Zone)
 title.papua_new_guinea=Papua New Guinea
 title.paraguay=Paraguay
 title.peru=Peru
 title.philippines=Philippines
-title.pitcairn=Pitcairn island
+title.pitcairn=Pitcairn,Henderson,Ducie and Oeno Islands
```

Comment:
I know in the spreadsheet there is no space following either of the commas, but would it be better if there were, given this is presumably a title for display?

Reply:
👍